css/index.css

```diff
@@ -126,21 +227,54 @@ p {
     padding-right: 0;
 }
 
-footer {
-    width: 100%;
-    background: black;
+/* ABOUT PAGE */
+
+/* Border Content */
+
+.about-page .border-content {
+    padding: 2%;
+    border-bottom: 1px dashed black;
 }
 
-footer nav {
-    width: 60%;
+/* Main Content */
+
+/* .about-page  */
+.main-content {
+    /* padding: 2% 0; */
+    margin: 2% 0;
     display: flex;
-    justify-content: space-between;
-    align-items: center;
-    padding: 20px 2%;
-    font-size: 14px;
+    Flex-flow: row wrap;
```

Comment:
While css properties are not case sensitive, its standard to keep them lower case.

Reply:
🤦